content/organizations/managing-user-access-to-your-organizations-repositories/repository-roles-for-an-organization.md

```diff
@@ -73,7 +73,9 @@ Some of the features listed below are limited to organizations using {% data var
 | Reopen issues they closed themselves | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
 | Have an issue assigned to them | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
 | Send pull requests from forks of the team's assigned repositories | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
-| Submit reviews on pull requests | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
+| [Submit reviews on pull requests](/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) | **✔️** | **✔️** | **✔️** | **✔️** | **✔️** |
+| [Submit reviews that affect a pull request's mergeability](/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews) | | | **✔️** | **✔️** | **✔️** |
```

Comment:
Not having the phrases `approve` or `require changes` means that searching for the things normal users experience doesn't work...

Reply:
@jsoref That's a fair point. How about `Approve or request changes on a pull request with required reviews`?

Reply:
I'll take that.